src/main/kotlin/uk/gov/justice/digital/hmpps/nomisprisonerapi/jpa/WorkFlow.kt

```diff
@@ -0,0 +1,50 @@
+package uk.gov.justice.digital.hmpps.nomisprisonerapi.jpa
+
+import jakarta.persistence.CascadeType
+import jakarta.persistence.Column
+import jakarta.persistence.DiscriminatorColumn
+import jakarta.persistence.Entity
+import jakarta.persistence.FetchType
+import jakarta.persistence.GeneratedValue
+import jakarta.persistence.Id
+import jakarta.persistence.Inheritance
+import jakarta.persistence.OneToMany
+import jakarta.persistence.SequenceGenerator
+import org.hibernate.Hibernate
+import org.hibernate.annotations.Generated
+import java.time.LocalDateTime
+
+@Entity(name = "WORK_FLOWS")
+@DiscriminatorColumn(name = "OBJECT_CODE")
+@Inheritance
+abstract class WorkFlow(
```

Comment:
Table is overloaded to point back to different entities based on the OBJECT_CODE